pkg/api/validation/validation.go

```diff
@@ -1547,12 +1646,12 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 func validateResourceName(value string, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if !validation.IsQualifiedName(value) {
-		return append(allErrs, field.Invalid(fldPath, value, "resource typename: "+qualifiedNameErrorMsg))
+		return append(allErrs, field.Invalid(fldPath, "", qualifiedNameErrorMsg))
 	}
 
 	if len(strings.Split(value, "/")) == 1 {
 		if !api.IsStandardResourceName(value) {
-			return append(allErrs, field.Invalid(fldPath, value, "is neither a standard resource type nor is fully qualified"))
+			return append(allErrs, field.Invalid(fldPath, "", "must be a standard resource type or fully qualified"))
```

Comment:
My mistake, I thought it was the value, but the error is the key.  But 'value' here is the key :)